.github/workflows/benchmarks.yml

```
@@ -89,6 +86,8 @@ jobs:
 
       - uses: elastic/oblt-actions/google/auth@v1
 
+      - uses: elastic/oblt-actions/aws/auth@v1
```

Shall we add the below header?

```suggestion
      # ONLY for Elastic employees
      # NOTE: If you change the name of the GitHub workflow then it's required to change the definition in
      #.            https://github.com/elastic/oblt-infra/tree/main/conf/resources/repos/apm-server/02-benchmarks.tf
      - uses: elastic/oblt-actions/aws/auth@v1
```